src/m365/entra/commands/multitenant/multitenant-get.ts

```diff
@@ -0,0 +1,44 @@
+import { Logger } from '../../../../cli/Logger.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+
+interface MultitenantOrganization {
```

Comment:
Can we use this interface in other multitenant commands as well? In that case I suggest that we extract it from this file and put it in the multitenant folder.

Reply:
Definitely. The issue is that https://www.npmjs.com/package/@microsoft/microsoft-graph-types wasn't updated for several months. The latest version doesn't contain model for multitenant org.

Reply:
Does it contain the interface now?

Reply:
No, it doesn't.
I mean that the latest version 2.40 of https://www.npmjs.com/package/@microsoft/microsoft-graph-types was released 8 months ago.